lib/Models/CsvCatalogItem.ts

```diff
@@ -141,7 +143,19 @@ export default class CsvCatalogItem
       | undefined = this.strata.get(
       automaticTableStylesStratumName
     ) as TableAutomaticStylesStratum;
-    return automaticTableStylesStratum?.discreteTimes;
+    return automaticTableStylesStratum?.discreteTimes?.reduce?.(
```

Comment:
I'm wondering if we need to move this down the tree further so it's applicable for other catalog item types too such as WMS?

Reply:
see https://github.com/TerriaJS/terriajs/pull/4577#issuecomment-669086838